_posts/2017/2017-09-06-napalm-logs-released.md

```diff
@@ -0,0 +1,325 @@
+``ip``: the IP address the device the message was recevied from.
+
+``os``: is the name of the network operating system identified from the syslog
+message format.
+
+``timestamp`` is the UTC timestamp of wen the syslog message was generated, not
+when received from the device.
+
+## Where are these messages available
+
+These structured messages are then published over various systems such as ZeroMQ
+(used by default), Kafka, et al. Similarly to the *listener* interface, the
+*publisher* interface is equally pluggable and the community can always contribute
+to extend the list of available transport.
+
+The outgoing messages are binary serialized using [MessagePack](http://msgpack.org/)
+and can be consumed using multiple clients.
+
+Considering that the napalm-logs messages can be used for event-driven automation
+and eventually trigger completely automatic configuration changes, we must ensure
+the authenticity of the messages. For this reason napalm-logs has been designed
+with security in mind, the serialized data is encrypted and signed before
+being published. With that said, the user must also ensure the client connecting
+to napalm-logs' publisher interface is designed with the ability to decrypt and
+check the signature of the received messages. For more details on  how to easily
+implement the security on the client side, have a look at the
+[Client Authentication](http://napalm-logs.readthedocs.io/en/latest/authentication/index.html).
+
+## Installation
+
+The library is available on [PyPI](https://pypi.python.org/pypi) (Python Package
+Index), which is the official repository for third-party Python libraries:
+
+```bash
+$ pip install napalm-logs
+```
+
+Which will also install the napalm-logs binary. For more specific installation
+notes, see [this document](http://napalm-logs.readthedocs.io/en/latest/installation/index.html).
+
+## How to start napalm-logs
+
+In the [main documentation](http://napalm-logs.readthedocs.io/en/latest/index.html)
+we explain briefly the ways to get started. You first need to determine if you
+*really* want to disable security.
+
+The easiest example to start with is the following:
+
+```bash
+$ sudo napalm-logs --publisher cli --disable-security
+```
+
+The above is highly discouraged in production environments, as it disables
+the security and just prints the objects on the command line; this is however a
+good way to check that the process is running correctly and receiving syslog
+messages. By default it is listening on the standard syslog UDP port ``514``
+(hence the need to start it with ``sudo``). To use a different port, you can use
+the [``port`` configuration option](http://napalm-logs.readthedocs.io/en/latest/options/index.html#port).
+
+The list of configuration options can grow significantly, although they can be
+equally specified on the command line when starting the daemon, it is preferable
+to add them into a configuration file (default: ``/etc/napalm/logs``):
+
+```/etc/napalm/logs```
```

Comment:
Since there's no linebreak after opening and before the closing of the ``` characters here it could blend into the yaml one and cause strange formatting.

Reply:
I've replaces the ``` with `` - is this what you meant? There should be a new line after this...

Reply:
That should be fine too. What I meant was that after a ``` there should be a newline (or the desired language of the code block), then after the code block there should be a newline and then a ``` to close the section. Since you had ```/etc.. on the same line it bleed into the next section. Looks good now.